<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>视频分割工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="input_group">
      <property name="title">
       <string>输入设置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="source_layout">
         <item>
          <widget class="QLabel" name="source_label">
           <property name="text">
            <string>源视频目录：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="source_path"/>
         </item>
         <item>
          <widget class="QPushButton" name="source_button">
           <property name="text">
            <string>浏览...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="output_layout">
         <item>
          <widget class="QLabel" name="output_label">
           <property name="text">
            <string>输出文件夹：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="output_path"/>
         </item>
         <item>
          <widget class="QPushButton" name="output_button">
           <property name="text">
            <string>浏览...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="settings_group">
      <property name="title">
       <string>分割设置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="duration_label">
         <property name="text">
          <string>分割时长(秒)：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="duration_spinbox">
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>3600.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="progress_group">
      <property name="title">
       <string>进度</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QProgressBar" name="progress_bar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="log_text">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="button_layout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="start_button">
        <property name="text">
         <string>开始分割</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
</ui> 